crates/re_space_view_tensor/src/tensor_dimension_mapper.rs

```diff
@@ -248,9 +248,9 @@ pub fn dimension_mapping_ui(
 
                         let response = re_ui.visibility_toggle_button(ui, &mut selector.visible);
                         if selector.visible {
-                            response.on_hover_text("Hide selector ui from the Space View.")
+                            response.on_hover_text("Hide selector ui from the Space View")
```

Comment:
`Selector ui` seems like an internal term that shouldn't reach users. What is this piece of UI? Can we call it something else?

Reply:
Actually... I'm not sure I'm able to find any mention of that "Selector UI" thing anywhere... We might very well get away with rewriting that tooltip without giving a name to that UI part altogether.  "Show/Hide dimension slider"?

Reply:
Love that suggestion! Maybe capitalized as `Show/hide dimension slider`?

Reply:
(It's actually two separate strings, so that capitalisation is a non-issue.)